src/main/java/core/basesyntax/database/Storage.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.database;
+
+import core.basesyntax.model.Fruit;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    public static final Map<Fruit, Integer> fruitsStorage = new HashMap<>();
```

Comment:
Let's make it simpler. You can delete class Fruit
```suggestion
    public static final Map<String, Integer> fruitsStorage = new HashMap<>();
```

Reply:
ok, but reverse process (adding class Fruit) if it's needed would be more difficult.